packages/google_maps_flutter/google_maps_flutter/example/lib/place_marker.dart

```diff
@@ -53,10 +53,10 @@ class PlaceMarkerBodyState extends State<PlaceMarkerBody> {
     final Marker? tappedMarker = markers[markerId];
     if (tappedMarker != null) {
       setState(() {
-        if (markers.containsKey(markerId)) {
-          final Marker resetOld = markers[markerId]!
+        if (markers.containsKey(selectedMarker)) {
```

Comment:
A cleaner null-safe way to write this given the limitations on analyzing members would be:

```
final MarkerId? previousMarkerId = selectedMarker;
if (previousMarkerId != null && markers.containsKey(previousMarkerId)) {
  ...
}

Reply:
Done 👍